libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -819,6 +819,9 @@ static DNNReturnType execute_model_tf(const DNNModel *model, const char *input_n
         outputs[i].data = TF_TensorData(output_tensors[i]);
```

Comment:
don't need to mention: with 4 outputs,  But the
parsing of tensorflow outputs is not done.

Reply:
sorry, i pushed the incorrect code yestorday

Reply:
sorry, i pushed the incorrect code yesterday

Reply:
will review after the new code is pushed